checks/active.go

```diff
@@ -51,5 +51,8 @@ func IsActive(c checker.Checker) checker.CheckResult {
 	return checker.CheckResult{
 		Pass:       totalCommits >= numCommits,
 		Confidence: confidence,
+		Description: `This check tries to determine if the project is still "actively maintained". 
```

Comment:
i dont like the idea of adding description here, help url should be enough and we shouldnt duplicate the doc text which can get updated with time (and also will be detailed).

Reply:
For `sarif` description is required.SARIF is a standard format and is being used at the GitHub Security center. How do we deal with that?

Reply:
Then need to move all strings data in xml file and then fetch description from there. we can have two fields, summary and detailed description. summary shows by default. we shouldn't put these long strings in code.

Reply:
![image](https://user-images.githubusercontent.com/172697/112358946-2b5eeb00-8ca7-11eb-987d-c3be9076de3d.png)

Which is the summary? The above section?

Reply:
Basically i want https://github.com/ossf/scorecard/blob/main/checks.md to be converted into a machine readable format, with summary, description and remediation fields. And then we read them in code and show it (normal view-summary only or detailed view which shows description and remediation as well). i want to reduce deduplication of text in checks.md and also dont want to hardcode strings in source code.

Reply:
OK. I understand. But the issue is that if we were covert it to machine-readable then how do we provide the same information as markdown when someone visits the GitHub Repo. Ideally, we have a `web` frontend that can serve this data and can be the single source of truth.

Reply:
I am fine to remove checks.md for now. We can later add it, can you create an issue.

Reply:
OK. I am fine with that. Just an FYI... this is a chore and would take some time.

Reply:
This is required to get the SARIF results and only after that, we can publish the scorecard as a GitHub action.

Reply:
We need more contributors.

Reply:
We have a new member starting on Scorecard project, so we should get help soon. Can table this one for now.

Reply:
Sounds good. Thanks

Reply:
big +1 to machine readable format for checker descriptions. We need this info for consumers of scorecard data to show in their own UI.